workflows/sample/tasks/hifiasm_trio.wdl

```diff
@@ -245,27 +191,6 @@ workflow hifiasm_trio {
       pb_conda_image = pb_conda_image
   }
 
-  call hifiasm.asm_stats as asm_stats_p_ctg  {
-    input:
-      fasta_gz = bgzip_fasta_p_ctg.fasta_gz,
-      index = target.indexfile,
-      pb_conda_image = pb_conda_image
-  }
-
-  call hifiasm.asm_stats as asm_stats_p_utg  {
-    input:
-      fasta_gz = bgzip_fasta_p_utg.fasta_gz,
-      index = target.indexfile,
-      pb_conda_image = pb_conda_image
-  }
-
-  call hifiasm.asm_stats as asm_stats_r_utg  {
-    input:
-      fasta_gz = bgzip_fasta_r_utg.fasta_gz,
-      index = target.indexfile,
-      pb_conda_image = pb_conda_image
-  }
-
```

Comment:
We only really care about the assembly stats of the finished assembly hap1 and hap2.